SYCL/InlineAsm/asm_float_add.cpp

```diff
@@ -1,9 +1,7 @@
 // UNSUPPORTED: cuda
 // REQUIRES: gpu,linux
-// RUN: %clangxx -fsycl %s -DINLINE_ASM -o %t.out
```

Comment:
Note: `INLINE_ASM` were invented on purpose to be able to easily switch test to non-inline-asm mode to get asm which is generated by a compiler for desired test - this is useful during test development/debugging.

However, I do agree that launching non-inline asm versions doesn't introduce a lot of value from testing prospective (as tests are too simple and we already have similar tests elsewhere) and it should be still easy to switch to non-inline-asm mode (add `0 && ` to current ifdef), so I'm ok with removing of this define